app/views/partials/theme/simple/dashboard/posts/index.liquid

```diff
@@ -1,69 +1,62 @@
-<div class="w-full mx-auto">
-  <div class="mb-5 text-right">
-    <a href="/dashboard/posts/new" class="button">
-      {{ 'app.models.posts.add' | t }}
-    </a>
-  </div>
-  <div class="my-6 bg-white rounded shadow-md">
-    {% if posts.results.size > 0 %}
-      <table class="w-full text-left border-collapse">
```

Comment:
Looks we have to change our generator to use `div` instead of `table` for `index.liquid` page.

Reply:
Tables are far harder to style responsively, so the DIVs, but I will try to make a partial that will render those based on the object/array passed to them and then we could change the generator to use that partial.